src/arena.py

```diff
@@ -1 +1,33 @@
-WORKS = True
+import json
+import random
+
+def calculate_main_damage(damage_pattern):
+	damage_factors = damage_pattern.split('d')
+	multiplier = int(damage_factors[0])
+	damage_range = int(damage_factors[1])
+	damage = 0;
+	for index in range(1, multiplier):
```

Comment:
`range(1, multiplier)` produces list of `multiplier-1` elements because `range` produces collection of numbers starting from one number (usually zero) and ending one before the other, so that the result list is: `[start, start+1, start+2, ..., end-1]` and has length `end-start`. Please use just `range(multiplier)`.
Also, as you don't use the `index` variable it's good to change its name to `__` - it's a common practice of marking unused variables (if there is no other way to remove it).